### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bsudekum**
Few minor updates. I removed `Installing Mapbox Core Navigation` because if anything is is confusing to a new user. 

/cc @mapbox/navigation-ios

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,8 +108,7 @@ class CustomStyle: DayStyle {
 
     override func apply() {
         super.apply()
-        ManeuverView.appearance().backgroundColor = .darkGray
-        RouteTableViewHeaderView.appearance().backgroundColor = .darkGray
+        BottomBannerView.appearance().backgroundColor = .orange
```
Comment (frederoni) [inline README.md]: Would it make sense to add a link to DayStyle.swift#master to showcase all styling capabilities w/o having to update this readme constantly?

Comment (JThramer) [reply README.md]: Yes, that's a great idea.

Comment (bsudekum) [reply README.md]: Added link in https://github.com/mapbox/mapbox-navigation-ios/pull/923/commits/259a294a5306eda3ba38da4e11fc64c50b006482. Let's keep this one line though so the user gets a feeling for how the API looks.

File: README.md [hunk 2]
```diff
@@ -153,33 +152,6 @@ If you need additional flexibility, you can use the following building blocks to
 * [OSRM Text Instructions for Swift](https://github.com/Project-OSRM/osrm-text-instructions.swift/) (also compatible with macOS, tvOS, and watchOS)
   * Localized guidance instructions
 
-### Installing Mapbox Core Navigation
```
Comment (1ec5) [inline README.md]: How about moving this content to a separate Markdown file? MapboxNavigation isn’t a one-size-fits-all solution, so we should have something linkable for situations when a developer finds Core Navigation to be more practical.

File: README.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ Mapbox Navigation requires a few additions to your `Info.plist`. Be sure to sign
 
 #### Styling
 
-You can customize the appearance in order to blend in with the rest of your app.
+You can customize the appearance in order to blend in with the rest of your app. Checkout [`DayStye.swift`](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/DayStyle.swift) for all styleable elements.
```
Comment (frederoni) [inline README.md]: Typo: s/DayStye/DayStyle

File: README.md [hunk 4]
```diff
@@ -72,9 +72,7 @@ import MapboxNavigation
 let origin = Waypoint(coordinate: CLLocationCoordinate2D(latitude: 38.9131752, longitude: -77.0324047), name: "Mapbox")
 let destination = Waypoint(coordinate: CLLocationCoordinate2D(latitude: 38.8977, longitude: -77.0365), name: "White House")
 
-let options = RouteOptions(forNavigationWithWaypoints: [origin, destination])
-options.routeShapeResolution = .full
-options.includesSteps = true
+let options = RouteNavigationOptions(forNavigationWithWaypoints: [origin, destination])
```
Comment (JThramer) [inline README.md]: Did you mean `NavigationRouteOptions`?

**Comment by JThramer** [comment]
Ah damn, didn't mean to do that. Ah well, yeah, you might want to fix that typo. Otherwise looks good.

File: README.md (hunk 4 above)
Comment (bsudekum) [reply README.md]: Oh yep

**Pull request merged**